packages/astro/e2e/view-transitions.test.js

```diff
@@ -413,10 +413,7 @@ test.describe('View Transitions', () => {
 		await expect(locator).toBeInViewport();
 
 		// Scroll back to top
-		// back returns immediately, but we need to wait for navigate() to complete
-		const waitForReady = page.waitForEvent('console');
 		await page.goBack();
-		await waitForReady;
 		locator = page.locator('#longpage');
 		await expect(locator).toBeInViewport();
```

Comment:
We should keep this test for now to test for current behavior

Reply:
Yep, agreed

Reply:
Does that mean adding prop to <ViewTransitions />?

Reply:
We fix the bug without adding a prop. We know how to introduce a compatibility toggle if this change would raise problems. But I guess people would rather be happy to get rid of the bug than to keep the status quo, if they care at all about this detail.